docs/msi_v2/guidance_for_sdks_consuming_msal.md

```diff
@@ -2,61 +2,30 @@
-- Secure token acquisition by enabling **mTLS authentication**.
-- **Reusable `HttpClient` instances** to prevent socket exhaustion.
-- Optimized **Azure SDK integration** for managing identities securely.
-
-### **Interface Definition**
-
-```csharp
-public interface IMsalMtlsHttpClientFactory : IMsalHttpClientFactory
-{
-    /// <summary>
-    /// Returns an HttpClient configured with a certificate for mutual TLS authentication.
-    /// </summary>
-    /// <param name="x509Certificate2">The certificate to be used for MTLS authentication.</param>
-    /// <returns>An HttpClient instance configured with the specified certificate.</returns>
-    HttpClient GetHttpClient(X509Certificate2 x509Certificate2);
-}
-```
-
 ## **Binding Certificate**
 
-SDKs customizing the httpclient factory will need a way to get the binding certificate for MTLS authentication. MSAL exposes a few different APIs to help SDKs manage certificates.
+SDKs customizing the httpclient factory will continue to use the old `IMtlsHttpClientFactory` interface. MSAL will use the customized httpclient factory to, 
 
-- Retrieving existing certificates from the OS store or memory.
-- Notifying SDKs when a certificate is updated via the CertificateRefreshed event.
+- call into `platformmetadata` endpoint to form the CSR.
+- Use the CSR and call into `issuecredential` endpoint to get the Certificate.
+- Once. MSAL acquires a certificate, MSAL will will call the mTLS endpoint using this certificate. MSAL will not use the customized factory for this call. 
 
-| API Name                         | Purpose                                                                            |
-|----------------------------------|------------------------------------------------------------------------------------|
-| `GetManagedIdentitySourceAsync()`| Will expose the MSI Source including the new `IMDSV2` source                       |
-| `GetBindingCertificate()`        | Helper method to get the binding certificate when source is `IMDSV2`.              |
-| `BindingCertificateRefreshed`    | Event to notify SDKs when the binding certificate is updated.                      |
+| API Name                             | Purpose                                                                            |
+|--------------------------------------|------------------------------------------------------------------------------------|
+| `GetManagedIdentitySourceAsync()`    | Will expose the MSI Source including the new `IMDSV2` source                       |
+| `BindingCertificateRefreshed`        | Event to notify SDKs when the binding certificate is updated.                      |
+| `IsPopSupported()`                   | Helper method to check if POP is supported.                                        |
+| `IsCaeSupported()`                   | Helper method to check if CAE is supported.                                        |
```

Comment:
When will CAE not be supported? On CloudShell and Arc? IMO this is not needed, we should just document.